backend/src/express.ts

```diff
@@ -0,0 +1,27 @@
+import express, {Request, Response } from 'express';
+import cookieParser from 'cookie-parser';
+import compression from 'compression';
+import morgan from 'morgan';
+import cors from 'cors';
+import apiRouter from './routes/api.router';
+// import healthRoute from './routes/health';
```

Comment:
Currently you have your health endpoint being controlled by `api.router.ts` You may have forgotten to rename it !